nmpolicy/internal/capture/capture_entry_test.go

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2021 NMPolicy Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *	  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package capture_test
+
+import (
+	"testing"
+
+	assert "github.com/stretchr/testify/require"
+
+	"github.com/nmstate/nmpolicy/nmpolicy/internal/capture"
+	"github.com/nmstate/nmpolicy/nmpolicy/types"
+)
+
+func TestCaptureEntry(t *testing.T) {
+	t.Run("test CaptureEntry", func(t *testing.T) {
+		testResolveCaptureEntryPathSuccess(t)
+
+		testResolveCaptureEntryPathWithLexFailure(t)
+		testResolveCaptureEntryPathWithParseFailure(t)
+		testResolveCaptureEntryPathWithResolveFailure(t)
+	})
+}
+
+func testResolveCaptureEntryPathSuccess(t *testing.T) {
+	t.Run("ResolveCaptureEntryPath success", func(t *testing.T) {
+		capturedStates := map[string]types.CaptureState{}
+		captureEntry, err := capture.NewCaptureEntryWithLexerParserResolver(capturedStates,
```

Comment:
Please introduce a similar method to `capture_test->captureResolverWithDefaultStubs() ` (e.g `captureEntryWithDefaultStubs`)